### PR TITLE
Removed counter and temp variables

### PR DESCRIPTION
**Pull request by GeetikaBatra**

### CODE REVIEW

**Comment by msrb** [comment]
[test]

**Comment by invincibleJai** [comment]
[test]

File: utils.py [hunk 1]
```diff
@@ -257,7 +253,7 @@ def service_get(company):
                     for service in v:
                         for i_k, i_v in service.items():
                             if (i_k == 'name'):
-                                if (i_v == 'service-' + company + '-' + counter + temp):
```
Comment (tisnik) [inline utils.py]: (minor: in Python you don't have to use parenthesis in if statement)

File: utils.py [hunk 2]
```diff
@@ -257,7 +253,7 @@ def service_get(company):
...
+                                if (i_v == 'service-' + company):
                                     print('Here isnide service')
```
Comment (tisnik) [inline utils.py]: (minor: typo isnide/inside)

Comment (GeetikaBatra) [reply utils.py]: @tisnik, Fixed it in this https://github.com/fabric8-analytics/f8a-3scale-connect-api/pull/10/commits/5694df15d25b5cc0d90a69c423c91f79ed5f5411

**Comment by invincibleJai** [comment]
@arunkumars08 does this look good ?

**Pull request merged**